dashboard/app/helpers/levels_helper.rb

```diff
@@ -180,7 +180,7 @@ def app_options
     if AuthoredHintViewRequest.enabled?
       view_options(authored_hint_view_requests_url: authored_hint_view_requests_path(format: :json))
       if current_user && @script
-        view_options(authored_hints_used_ids: Set.new(AuthoredHintViewRequest.hints_used(current_user.id, @script.id, @level.id).map(&:hint_id)))
+        view_options(authored_hints_used_ids: Set.new(AuthoredHintViewRequest.hints_used(current_user.id, @script.id, @level.id).pluck(:hint_id)))
```

Comment:
Is this in a `Set.new` to make sure the list is unique? It could isntead be `.pluck(:hint_id).uniq`